policyengine_us/variables/gov/states/oh/tax/income/credits/oh_retirement_income_credits.py

```diff
@@ -0,0 +1,31 @@
+from policyengine_us.model_api import *
+
+
+class oh_retirement_income_credits(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Ohio Retirement Income Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://codes.ohio.gov/ohio-revised-code/section-5747.055"
+    defined_for = StateCode.OH
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.oh.tax.income.credits.retirement_income
+
+        person = tax_unit.members
+        pension_income = sum(person("pension_income", period))
+
+        agi = tax_unit("oh_agi", period)
+        has_not_taken_lump_sum_distribution = person(
+            "oh_has_not_taken_oh_lump_sum_credits", period
+        )
+
+        eligible = agi < p.agi_cap and pension_income > 0
```

Comment:
dont think we need the pension income eligibility part as the credit amount would be 0 below $500 anyways